resources/data/arm_emb-8-2019-q3.json

```diff
@@ -0,0 +1,71 @@
+{
+    "type": "Recipe",
+    "title": "Install the toolchain",
+    "tool": "arm_emb-8-2019-q3",
```

Comment:
I suggest to rename the file not to indicate the gcc version, e.g. `arm_emb.json`. Remember the reference in `course_zephyr.json`.